docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Branch Review
+sidebar_position: 35
+---
+
+Cypress Branch Review allows you to quickly identify the impact a pull request might have on your test suite in a single view. Compare which tests are failing, flaky, pending, added, or modified between the source and base branches.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review.jpg"
+  alt="Cypress Branch Review"
+/>
+
+:::info
+
+Cypress Branch Review is currently only available for GitHub integrations. We are working on adding support for GitLab and Bitbucket soon.
+
+:::
+
+## Getting Started
+
+Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub.
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have an open pull request.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+/>
+
+## Branch Details
+
+### View Options
+
+Once the GitHub integration is configured for a project, "Branch Review" is the default view of the **Branches** tab. Otherwise, a list of "Runs" filtered by this branch will default with tips for how to enable and maximize this feature.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  alt="Displaying different views for branches"
+/>
+
+### Review Header
+
+At the left side of the header is the Git commit message, PR selector, PR status and base and target branch indicators. If there are multiple pull requests open for the same branch, you can select the pull request you want to review from the dropdown to the right of the commit message.
+
+The right side of the header includes the Cypress Cloud test run number and test summary. Clicking the test run number will link you directly to the [test overview](/guides/cloud/recorded-runs#Overview-tab).
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-header.jpg"
+  alt="Branch Review Header"
+/>
+
+### Review Test Status
+
+When on the review screen, you will see tabs Failures, Flaky, Pending, Added, and Modified. Each tab will show you the tests that fall into that category. You can click on a test to view the test details.
```

Comment:
we should also document the subsections of "new"/"existing"/"previous/resolved" – these are probably one of the most confusing parts of the feature, what falls under each